source/message/OP_MSG.rst

```diff
@@ -51,7 +51,9 @@ Usage
 -----
 
 ``OP_MSG`` is only available in MongoDB 3.6 (``maxWireVersion >= 6``) and later.
-MongoDB drivers MUST continue to perform the MongoDB Handshake using ``OP_QUERY``
+MongoDB drivers SHOULD perform the MongoDB handshake using ``OP_MSG`` if an API
+version was declared on the client, but MAY decide to use ``OP_QUERY``. If no
+API version was declared, drivers MUST perform the handshake using ``OP_QUERY``
```

Comment:
In practice we are developing drivers (e.g. Rust) that only support 3.6+ servers.  Let's clarify here that OP_QUERY is only required for drivers that support 3.4-

Reply:
Clarified the wording.

Reply:
Clarified the wording.